Library/ViewModels/ManageViewPledgeViewModel.swift

```diff
@@ -3,15 +3,25 @@ import Prelude
 import ReactiveExtensions
 import ReactiveSwift
 
+public enum ActionSheetMenuOption: CaseIterable {
```

Comment:
This naming seems like it's very high up the domain level (maybe even on a UIKit level)...how about we make it more local `ManagePledgeAlertAction`? This naming makes it explicitly clear that this is `Managing pledge` (since it's `public`) and also makes it clearer to UIKit component we're setting it on to `UIAlertAction`. What do you think?

Reply:
I don't feel strongly about the `MenuOption` naming, just wanted to give another suggestion.